packages/orchestration/src/exos/stakingAccountKit.js

```diff
@@ -116,6 +161,7 @@ export const prepareStakingAccountKit = (baggage, makeRecorderKit, zcf) => {
               Any.toJSON(
                 MsgDelegate.toProtoMsg({
                   delegatorAddress,
+                  // @ts-expect-error Type 'CosmosValidatorAddress' is not assignable to type 'string'.
```

Comment:
so how is this supposed to work?

Reply:
Good question, advice welcome

Reply:
I could add a ref for #9162 to the comment

Reply:
This was a brain fart, now fixed